pxtlib/tutorial.ts

```diff
@@ -53,7 +53,7 @@ namespace pxt.tutorial {
     }
 
     function parseTutorialSteps(tutorialmd: string): TutorialStepInfo[] {
-        const filterblocksRegex = /```(filterblocks)\s*\n([\s\S]*?)\n```/gmi;
+        const hiddenSnippetRegex = /```(filterblocks|package)\s*\n([\s\S]*?)\n```/gmi;
```

Comment:
There is a number of those. See tutorial css

Reply:
so I'd want to add config and ghost here from the link below to this regex, correct? Looks like that's all the ones I see to avoid in https://makecode.com/writing-docs/macros